File: RQ1_Prompt_Elements/data/scenario2/id_98.java
package scenario2;


import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class CountUpper {
    /**
     * * Given a string s, count the number of uppercase vowels in even indices.
     * 
     */
    public static int countUpper(String s) {
        int result = 0;
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if (i % 2 == 0 && Character.isUpperCase(c) && "AEIOU".indexOf(c) >= 0) {
                result += 1;
            }
        }
        return result;
    }

}

File: HumanEvalPython/scenario2/id_98.py

def count_upper(s):
    """
    Given a string s, count the number of uppercase vowels in even indices.
    
    
    """

    count = 0
    for i in range(0,len(s),2):
        if s[i] in "AEIOU":
            count += 1
    return count
